internal/mode/static/handler.go

```diff
@@ -68,6 +75,18 @@ func newEventHandlerImpl(cfg eventHandlerConfig) *eventHandlerImpl {
 }
 
 func (h *eventHandlerImpl) HandleEventBatch(ctx context.Context, batch events.EventBatch) {
+	start := time.Now()
+	h.cfg.logger.V(1).Info("Started processing event batch")
```

Comment:
Sure, that makes sense. Should this ID be a number we increment, like config version, or a generated UUID?

Reply:
sequential makes sense to me, because it is simple - and I can't see the need for a unique UUID

Reply:
Added a batch ID to the event loop and injected the logger with the batch ID into the HandleEventBatch function.

Reply:
```
{"level":"info","ts":"2023-10-12T17:46:24Z","logger":"eventLoop.eventHandler","msg":"Handling events from the batch","batchID":7,"total":1}
{"level":"debug","ts":"2023-10-12T17:46:24Z","logger":"eventLoop.eventHandler","msg":"Started processing event batch","batchID":7}
{"level":"info","ts":"2023-10-12T17:46:24Z","logger":"statusUpdater","msg":"Updating Nginx Gateway status"}
{"level":"info","ts":"2023-10-12T17:46:24Z","logger":"eventLoop.eventHandler","msg":"Handling events didn't result into NGINX configuration changes","batchID":7}
{"level":"debug","ts":"2023-10-12T17:46:24Z","logger":"eventLoop.eventHandler","msg":"Finished processing event batch","batchID":7,"duration":"20.3554ms"}
{"level":"info","ts":"2023-10-12T17:46:24Z","logger":"eventLoop.eventHandler","msg":"Finished handling the batch","batchID":7}
```

I wasn't sure about injecting the logger into the status update function since it's also called asynchronously by the leader elector.